src/containers/shared/utils.js

```diff
@@ -33,6 +33,7 @@ export const CURRENCY_REGEX =
   /^[a-zA-Z0-9]{3,}\.r[rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz]{27,35}$/
 export const FULL_CURRENCY_REGEX =
   /^[0-9A-Fa-f]{40}\.r[rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz]{27,35}$/
+export const VALIDATORS_REGEX = /^n[9H][0-9A-Za-z]{50}$/
```

Comment:
@mDuo13 is this a good regex for determining if a key is a validator?  Are there other rules?

Reply:
I'm not sure where this regex is from, but I believe all validation public keys start with `n9` or `nH` because of the way the [base58 encoding](https://xrpl.org/base58-encodings.html) works - using the predefined prefix byte and the XRPL alphabet ([implementation](https://github.com/XRPLF/xrpl.js/blob/76b73e16a97e1a371261b462ee1a24f1c01dbb0c/packages/ripple-address-codec/src/xrp-codec.ts#L239-L242)).

I think we can go with this regex, and update it in the future if needed.